### DISCUSSION TITLE
Question about pruning

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5727] [Q&A] [answered]
I have a question about the Allow Message Archiving checkbox under the channel pruning options.  They manual states:  If checked, messages in this channel are archived on your system before being pruned from MC. 

If messages are pruned, how are these archived messages accessible? 

I'm trying to set up a mirth instance that won't save any messages to disk (or db). I just need it to be able to handle processing large files, with embedded rtf files, and move them from point a to point b with no or minimal transformations. Disk space is limited so I need to reduce its usage as much as possible.

**Comment by pacmano1** [comment]
They are written to disk.   Look at the global pruner settings.

**Comment by jonbartels** [comment, marked as answer]
>  Disk space is limited so I need to reduce its usage as much as possible.

This is a broader question. Asking about pruning and archiving is a good start. Read up on "Remove Content on Completion" this removes a message as soon as it is SENT and not errored. 

You also want to use [message attachments ](https://forums.mirthproject.io/articles/175046-using-attachments) and look at the [message retention settings ](https://docs.nextgen.com/bundle/Mirth_User_Guide_40/page/connect/connect/topics/c_Message_Storage_Settings_connect_ug.html)
